From 9cd4f409e6d78aad4b70ef16810526ebc6bc24de Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 9 May 2023 14:25:10 +0100
Subject: [PATCH] overlays: bcm2711: Remove I/O pinctrl references

Now that the base bcm2711 base dts files give the added I/O interfaces
references to the default pinctrl nodes, remove the same from their
respective overlays.

See: https://github.com/raspberrypi/linux/pull/5443

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/i2c3-overlay.dts     | 2 --
 arch/arm/boot/dts/overlays/i2c4-overlay.dts     | 2 --
 arch/arm/boot/dts/overlays/i2c5-overlay.dts     | 2 --
 arch/arm/boot/dts/overlays/i2c6-overlay.dts     | 2 --
 arch/arm/boot/dts/overlays/spi3-1cs-overlay.dts | 2 --
 arch/arm/boot/dts/overlays/spi3-2cs-overlay.dts | 2 --
 arch/arm/boot/dts/overlays/spi4-1cs-overlay.dts | 2 --
 arch/arm/boot/dts/overlays/spi4-2cs-overlay.dts | 2 --
 arch/arm/boot/dts/overlays/spi5-1cs-overlay.dts | 2 --
 arch/arm/boot/dts/overlays/spi5-2cs-overlay.dts | 2 --
 arch/arm/boot/dts/overlays/spi6-1cs-overlay.dts | 2 --
 arch/arm/boot/dts/overlays/spi6-2cs-overlay.dts | 2 --
 arch/arm/boot/dts/overlays/uart2-overlay.dts    | 2 --
 arch/arm/boot/dts/overlays/uart3-overlay.dts    | 2 --
 arch/arm/boot/dts/overlays/uart4-overlay.dts    | 2 --
 arch/arm/boot/dts/overlays/uart5-overlay.dts    | 2 --
 16 files changed, 32 deletions(-)

--- a/arch/arm/boot/dts/overlays/i2c3-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c3-overlay.dts
@@ -8,8 +8,6 @@
 		target = <&i2c3>;
 		frag0: __overlay__ {
 			status = "okay";
-			pinctrl-names = "default";
-			pinctrl-0 = <&i2c3_pins>;
 			clock-frequency = <100000>;
 		};
 	};
--- a/arch/arm/boot/dts/overlays/i2c4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c4-overlay.dts
@@ -8,8 +8,6 @@
 		target = <&i2c4>;
 		frag0: __overlay__ {
 			status = "okay";
-			pinctrl-names = "default";
-			pinctrl-0 = <&i2c4_pins>;
 			clock-frequency = <100000>;
 		};
 	};
--- a/arch/arm/boot/dts/overlays/i2c5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c5-overlay.dts
@@ -8,8 +8,6 @@
 		target = <&i2c5>;
 		frag0: __overlay__ {
 			status = "okay";
-			pinctrl-names = "default";
-			pinctrl-0 = <&i2c5_pins>;
 			clock-frequency = <100000>;
 		};
 	};
--- a/arch/arm/boot/dts/overlays/i2c6-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c6-overlay.dts
@@ -8,8 +8,6 @@
 		target = <&i2c6>;
 		frag0: __overlay__ {
 			status = "okay";
-			pinctrl-names = "default";
-			pinctrl-0 = <&i2c6_pins>;
 			clock-frequency = <100000>;
 		};
 	};
--- a/arch/arm/boot/dts/overlays/spi3-1cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi3-1cs-overlay.dts
@@ -20,8 +20,6 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pinctrl-names = "default";
-		        pinctrl-0 = <&spi3_pins &spi3_cs_pins>;
 			cs-gpios = <&gpio 0 1>;
 			status = "okay";
 
--- a/arch/arm/boot/dts/overlays/spi3-2cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi3-2cs-overlay.dts
@@ -20,8 +20,6 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pinctrl-names = "default";
-		        pinctrl-0 = <&spi3_pins &spi3_cs_pins>;
 			cs-gpios = <&gpio 0 1>, <&gpio 24 1>;
 			status = "okay";
 
--- a/arch/arm/boot/dts/overlays/spi4-1cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi4-1cs-overlay.dts
@@ -20,8 +20,6 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pinctrl-names = "default";
-		        pinctrl-0 = <&spi4_pins &spi4_cs_pins>;
 			cs-gpios = <&gpio 4 1>;
 			status = "okay";
 
--- a/arch/arm/boot/dts/overlays/spi4-2cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi4-2cs-overlay.dts
@@ -20,8 +20,6 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pinctrl-names = "default";
-		        pinctrl-0 = <&spi4_pins &spi4_cs_pins>;
 			cs-gpios = <&gpio 4 1>, <&gpio 25 1>;
 			status = "okay";
 
--- a/arch/arm/boot/dts/overlays/spi5-1cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi5-1cs-overlay.dts
@@ -20,8 +20,6 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pinctrl-names = "default";
-		        pinctrl-0 = <&spi5_pins &spi5_cs_pins>;
 			cs-gpios = <&gpio 12 1>;
 			status = "okay";
 
--- a/arch/arm/boot/dts/overlays/spi5-2cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi5-2cs-overlay.dts
@@ -20,8 +20,6 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pinctrl-names = "default";
-		        pinctrl-0 = <&spi5_pins &spi5_cs_pins>;
 			cs-gpios = <&gpio 12 1>, <&gpio 26 1>;
 			status = "okay";
 
--- a/arch/arm/boot/dts/overlays/spi6-1cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi6-1cs-overlay.dts
@@ -20,8 +20,6 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pinctrl-names = "default";
-		        pinctrl-0 = <&spi6_pins &spi6_cs_pins>;
 			cs-gpios = <&gpio 18 1>;
 			status = "okay";
 
--- a/arch/arm/boot/dts/overlays/spi6-2cs-overlay.dts
+++ b/arch/arm/boot/dts/overlays/spi6-2cs-overlay.dts
@@ -20,8 +20,6 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 
-			pinctrl-names = "default";
-		        pinctrl-0 = <&spi6_pins &spi6_cs_pins>;
 			cs-gpios = <&gpio 18 1>, <&gpio 27 1>;
 			status = "okay";
 
--- a/arch/arm/boot/dts/overlays/uart2-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart2-overlay.dts
@@ -7,8 +7,6 @@
 	fragment@0 {
 		target = <&uart2>;
 		__overlay__ {
-			pinctrl-names = "default";
-			pinctrl-0 = <&uart2_pins>;
 			status = "okay";
 		};
 	};
--- a/arch/arm/boot/dts/overlays/uart3-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart3-overlay.dts
@@ -7,8 +7,6 @@
 	fragment@0 {
 		target = <&uart3>;
 		__overlay__ {
-			pinctrl-names = "default";
-			pinctrl-0 = <&uart3_pins>;
 			status = "okay";
 		};
 	};
--- a/arch/arm/boot/dts/overlays/uart4-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart4-overlay.dts
@@ -7,8 +7,6 @@
 	fragment@0 {
 		target = <&uart4>;
 		__overlay__ {
-			pinctrl-names = "default";
-			pinctrl-0 = <&uart4_pins>;
 			status = "okay";
 		};
 	};
--- a/arch/arm/boot/dts/overlays/uart5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart5-overlay.dts
@@ -7,8 +7,6 @@
 	fragment@0 {
 		target = <&uart5>;
 		__overlay__ {
-			pinctrl-names = "default";
-			pinctrl-0 = <&uart5_pins>;
 			status = "okay";
 		};
 	};
